File: Clases/Unidad02_POO/Publicaciones_v02/src/main/java/publicaciones_v02/Libro.java
package publicaciones_v02;

/*
 * @author nickc
 */

public class Libro extends Publicacion{
// Atributos
    private String edicion;
    private String lugar;
    private String editorial;
// Metodos
    // Constructor
    public Libro(String nombre, String autores[], Integer cantidad_autores, Integer año, 
            String edicion, String lugar, String editorial){
        super(nombre, autores, cantidad_autores, año);
        this.editorial = editorial;
        this.lugar = lugar;
        this.editorial = editorial;
    }
    public Libro(String nombre, String autor, Integer año, String edicion, String lugar,
            String editorial){
        super(nombre, autor, año);
        this.editorial = editorial;
        this.lugar = lugar;
        this.editorial = editorial;
    }
}

File: Clases/Unidad02_POO/C#/Publicaciones_v02/Publicaciones_v02/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Publicaciones_v02
{
    internal class Libro: Publicacion
    {
    // Atributos
        private string edicion;
        private string lugar;
        private string editorial;
    // Metodos
        // Constructor
        public Libro(String nombre, String[] autores, int cantidad_autores, int año,
            String edicion, String lugar, String editorial):
            base(nombre, autores, cantidad_autores, año)
        {
            this.edicion = edicion;
            this.lugar = lugar;
            this.editorial = editorial;
        }
        public Libro(String nombre, String autor, int año, String edicion, String lugar,
                String editorial):
             base(nombre, autor, año)
        {
            this.edicion = edicion;
            this.lugar = lugar;
            this.editorial = editorial;
        }
    }
}
